geode-redis/src/main/java/org/apache/geode/redis/internal/RedisCommandType.java

```diff
@@ -181,6 +181,7 @@
    ***************************************/
   // do not call these directly, only to be used in other commands
   INTERNALPTTL(null, INTERNAL, new ExactParameterRequirements(2)),
+  INTERNALTYPE(null, INTERNAL, new ExactParameterRequirements(2)),
   INTERNALSMEMBERS(null, INTERNAL, new ExactParameterRequirements(3)),
```

Comment:
Is it weird that a user would be able to call these from the CLI and would get different error messages if they did (like "wrong number of arguments" instead of "unknown command" error?  Not sure why they would and maybe it doesn't really matter.  I guess I'm just wondering if there's a way of preventing that or some other way of doing this.

Reply:
I have a story to address this. Also, would like to hear your thoughts on how better to address the problem we're trying to fix with these 'internal' commands.